### PR TITLE
🎨 Fix English in an error

### PR DESCRIPTION
**Pull request by zeenix**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -188,7 +188,7 @@ fn install_hook(color_scheme: ColorScheme) -> Result<(), Box<dyn Error>> {
         Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {
             eprintln!(
                 "Failed to create `{HOOK_PATH}` as it already exists. \
-                Please either or remove it and re-run `gimoji -i` or \
+                Please either remove it and re-run `gimoji -i`, or \
                 add the following commandline to it:\n{content}",
```
Comment (poettering) [inline src/main.rs:192]: "command line"

Comment (zeenix) [reply src/main.rs:192]: or "command-line" according to [wikipedia](https://en.wikipedia.org/wiki/Command-line_interface).

File: src/main.rs [hunk 2]
```diff
@@ -188,8 +188,8 @@ fn install_hook(color_scheme: ColorScheme) -> Result<(), Box<dyn Error>> {
         Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {
             eprintln!(
                 "Failed to create `{HOOK_PATH}` as it already exists. \
-                Please either or remove it and re-run `gimoji -i` or \
-                add the following commandline to it:\n{content}",
+                Please either remove it and re-run `gimoji -i`, or \
+                add the following command-line to it:\n{content}",
```
Comment (poettering) [inline src/main.rs:192]: that's german.

command line is english

Comment (zeenix) [reply src/main.rs:192]: https://en.wikipedia.org/wiki/Command-line_interface

Comment (zeenix) [reply src/main.rs:192]: that page on the `en` subdomain, has "command-line" all over it.

Comment (poettering) [reply src/main.rs:192]: it's "a command line", but "a command-line interface"

**Pull request merged**